definitions/procedures/puppet/enable_puppet.rb

```diff
@@ -0,0 +1,49 @@
+module Procedures::Puppet
+  class EnablePuppet < ForemanMaintain::Procedure
+    metadata do
+      description 'Enable Puppet feature'
```

Comment:
Just saying outloud that this starts to feel like scope creep and almost more like something the installer should have a native option for through hooks. Disable given it touches many things on the system feels better in foreman-maintain, this feels like just wrapping installer options. I don't know where to draw the line and this is not a blocker but I think it's worth pausing a moment to ensure this is what we intend.

Also, I am assuming this is being added just as a convenience other documenting how to do it?

Reply:
I had the same thought. I can't find the comment now but in the past I have thought about an `on_module_enable` / `on_module_disable` hook. IMHO there's at least a few options we have. But I agree it's more of a workaround in the wrong tool due to deadlines.

Reply:
Marking this resolved as enable is out of scope.